be/src/io/fs/file_writer.h

```diff
@@ -33,6 +33,7 @@ struct FileWriterOptions {
     bool write_file_cache = false;
     bool is_cold_data = false;
     int64_t file_cache_expiration = 0; // Absolute time
+    bool sync_file_data = true;        // weather flush data into storage system
```

Comment:
```suggestion
    bool sync_file_data = true;        // Whether flush data into storage system
```
另外比较建议把这个bool放到int64之前让struct内存布局更紧凑

Reply:
然后可以把"// Only affects remote file writers"删了